content/millennium/dstu2/scheduling/schedule.md

```diff
@@ -0,0 +1,107 @@
+---
+title: Schedule | DSTU 2 API
+---
+
+# Schedule
+
+* TOC
+{:toc}
+
+## Overview
+
+Schedule resources provide a container for time-slots that can be booked using an appointment. It provides the window of time (period) that slots are defined for and what type of appointments can be booked.
+<br/><br/>
+The following fields are returned if valued:
+
+* [Type](http://hl7.org/fhir/DSTU2/schedule-definitions.html#Schedule.type){:target="_blank"}
+* [Actor](http://hl7.org/fhir/DSTU2/schedule-definitions.html#Schedule.actor){:target="_blank"}
+* [Location Extension](#extensions)
+
+## Terminology Bindings
+
+<%= terminology_table(:schedule, :dstu2) %>
+
+## Extensions
+
+* [Location]
+
+### Custom Extensions
+
+All URLs for custom extensions are defined as `https://fhir-ehr.cerner.com/dstu2/StructureDefinition/{id}`
+
+ID             | Value\[x] Type    | Description
+---------------|-------------------|----------------------------------------------------------------------------------
+`location`      | [`Reference`]          | Reference to the location corresponding to this schedule, where the appointment can be booked.
+
+## Search
+
+Search for Schedules that meet supplied query parameters:
+
+    GET /Schedule?:parameters
+
+### Authorization Types
+
+<%= authorization_types(practitioner: true, patient: true, system: true) %>
+
+### Parameters
+
+ Name         | Required?                             | Type          | Description
+--------------|---------------------------------------|---------------|------------------------------------------------------------------------------------
+`_id`         | Y                                     | [`token`]     | The logical resource id associated with the resource.
+
+### Headers
+
+ <%= headers %>
+
+### Example
+
+#### Request
+
+    GET https://fhir-open.sandboxcerner.com/dstu2/0b8a0111-e8e6-4c26-a91c-5069cbc6b1ca/Schedule?_id=166627791-4048128-9331486-0
```

Comment:
In the worst case this should come out to exactly the character limit:
9223372036854775807-9223372036854775807-9223372036854775807-1495

It would require artificially boosting the sequence buckets for 3 different columns on the backend as well as having a scheduling build with full-day contiguous slots with 5 minute scheduling intervals.